eng/helix/content/Download.ps1

```diff
@@ -29,15 +29,16 @@ while($true) {
     }
     catch {
       Write-Host "Failed to download '$uri'"
-      Write-Error $_.Exception.Message -ErrorAction Continue
     }
 
     if (++$retries -le $maxRetries) {
+      Write-Warning $_.Exception.Message -ErrorAction Continue
```

Comment:
Have you tested to be sure `$_.Exception` is available outside the `catch`❔ I doubt it is. Might need to copy the message into a variable within the `catch` block.